src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/Common/CommonTypes.Windows.cs

```diff
@@ -2,12 +2,34 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#pragma warning disable CS0618 // Type or member is obsolete
+
 namespace System.Runtime.InteropServices.Tests.Common
 {
     [ComImport]
     [Guid("927971f5-0939-11d1-8be1-00c04fd8d503")]
     public interface IComImportObject { }
 
+    [ComImport]
+    [Guid("927971f5-0939-11d1-8be1-00c04fd8d503")]
```

Comment:
Guids are all identical. Use `uuidgen /c` from Visual Studio Developer Command prompt to generate guids.

Reply:
I changed the guids for the interfaces, but doing this with the classes causes test failures:
```
  ----- start 11:43:07.47 ===============  To repro directly: =====================================================
  pushd C:\Users\hughb\Documents\GitHub\corefx\bin\tests\System.Runtime.InteropServices.Tests\netcoreapp-Windows_NT-Deb
  ug-x64\
  call C:\Users\hughb\Documents\GitHub\corefx\bin\testhost\netcoreapp-Windows_NT-Debug-x64\\dotnet.exe xunit.console.ne
  tcore.exe System.Runtime.InteropServices.Tests.dll  -xml testResults.xml -notrait Benchmark=true -notrait category=no
  nnetcoreapptests -notrait category=nonwindowstests  -notrait category=OuterLoop -notrait category=failing
  popd
  ===========================================================================================================
  xUnit.net console test runner (64-bit .NET Core)
  Copyright (C) 2014 Outercurve Foundation.

  Discovering: System.Runtime.InteropServices.Tests
  Discovered:  System.Runtime.InteropServices.Tests
  Starting:    System.Runtime.InteropServices.Tests
     System.Runtime.InteropServices.Tests.IsComObjectTests.IsComObject_Windows_ReturnsExpected [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\Marshal\IsComObjectTests.Windows.cs(15,0): at System.Runtime.InteropServices.Tests.IsComObjectTests.IsComObject_Wi
  ndows_TestData()+MoveNext()
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Linq\src\System\Linq\Select.cs(135,0): at System.Linq.Enum
  erable.SelectEnumerableIterator`2.MoveNext()
     System.Runtime.InteropServices.Tests.GetComObjectDataTests.GetComObjectData_ValidObject_ReturnsExpected [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\Marshal\GetComObjectDataTests.Windows.cs(16,0): at System.Runtime.InteropServices.Tests.GetComObjectDataTests.GetC
  omObjectData_ValidObject_ReturnsExpected()
     System.Runtime.InteropServices.Tests.SetComObjectDataTests.SetComObjectData_NullValue_Sucesss [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\Marshal\SetComObjectDataTests.Windows.cs(32,0): at System.Runtime.InteropServices.Tests.SetComObjectDataTests.SetC
  omObjectData_NullValue_Sucesss()
     System.Runtime.InteropServices.Tests.SetComObjectDataTests.SetComObjectData_NonNullValue_Sucesss [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\Marshal\SetComObjectDataTests.Windows.cs(16,0): at System.Runtime.InteropServices.Tests.SetComObjectDataTests.SetC
  omObjectData_NonNullValue_Sucesss()
     System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHandler_TargetNotIConnectionIConnectionPointCo
  ntainer_ThrowsInvalidCastException [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\ComAwareEventInfoTests.Windows.cs(112,0): at System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventH
  andler_TargetNotIConnectionIConnectionPointContainer_ThrowsInvalidCastException()
     System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHandler_ComObjectWithMultipleComEventInterface
  Attribute_ThrowsAmbiguousMatchException [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\ComAwareEventInfoTests.Windows.cs(57,0): at System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHa
  ndler_ComObjectWithMultipleComEventInterfaceAttribute_ThrowsAmbiguousMatchException()
     System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHandler_NoSuchSourceTypeEventInterface_ThrowsA
  rgumentNullException [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\ComAwareEventInfoTests.Windows.cs(84,0): at System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHa
  ndler_NoSuchSourceTypeEventInterface_ThrowsArgumentNullException()
     System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHandler_NoDispIdAttribute_ThrowsInvalidOperati
  onException [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\ComAwareEventInfoTests.Windows.cs(101,0): at System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventH
  andler_NoDispIdAttribute_ThrowsInvalidOperationException()
     System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHandler_ComObjectWithoutComEventInterfaceAttri
  bute_ThrowsInvalidOperationException [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\ComAwareEventInfoTests.Windows.cs(32,0): at System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHa
  ndler_ComObjectWithoutComEventInterfaceAttribute_ThrowsInvalidOperationException()
     System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHandler_NullSourceTypeEventInterface_ThrowsNul
  lReferenceException [FAIL]
        System.Runtime.InteropServices.COMException : Retrieving the COM class factory for component with CLSID {838CEA
  E6-4C5E-4364-B2F8-B9024B35BC8E} failed due to the following error: 80040154 Class not registered (Exception from HRES
  ULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
        Stack Trace:
           C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime.InteropServices\tests\System\Runtime\InteropServic
  es\ComAwareEventInfoTests.Windows.cs(67,0): at System.Runtime.InteropServices.Tests.ComAwareEventInfoTests.AddEventHa
  ndler_NullSourceTypeEventInterface_ThrowsNullReferenceException()
  Finished:    System.Runtime.InteropServices.Tests
```